packages/desktop/views/dashboard/Sidebar.svelte

```diff
@@ -39,6 +38,7 @@
                       label: localize('tabs.collectibles'),
                       route: DashboardRoute.Collectibles,
                       onClick: openCollectibles,
+                      disabled: Boolean(!$selectedWallet?.mainAccountId),
```

Comment:
! will already give you a boolean
```suggestion
                      disabled: !$selectedWallet?.mainAccountId,
```

Reply:
same for the rest